mods/numbers/src/service/assertions.ts

```diff
@@ -1,16 +1,19 @@
 import { FonosterInvalidArgument } from "@fonoster/errors";
 import { NumbersPB } from "../client/numbers";
 import { IngressInfo } from "./protos/numbers_pb";
+import { phone } from 'phone'
 
 const isValidURL = (value: string) =>
   /^(?:\w+:)?\/\/([^\s\.]+\.\S{2}|localhost[\:?\d]*)\S*$/.test(value);
 
 export const assertIsE164 = (number: string) => {
-  if (!number) {
+  const validatedPhone = phone(number, { validateMobilePrefix: false })
+  if (!validatedPhone.isValid) {
     throw new FonosterInvalidArgument(
       "e164Number field must be a valid e164 value."
     );
   }
+  return validatedPhone.phoneNumber;
```

Comment:
This will return a valid number with plus sign and be used to send  to routr for provisioning, is this accepted? or should we remove plus sign before sending it.

Reply:
Yes. We want to normalize to always use `+`.